frontend/ui/widget/inputdialog.lua

```diff
@@ -347,7 +348,9 @@ function InputDialog:init()
                                     - vspan_after_input_text:getSize().h
                                     - buttons_container:getSize().h
                                     - keyboard_height
-        if self.fullscreen or text_height > available_height then
+                                    - (self.fullscreen and 0 or Size.padding.large) -- keep some padding around
```

Comment:
To keep current (good) behaviour everywhere, let's apply this new additional padding when `use_available_height` only.

Reply:
Or just don't add any.
It feels you added it so "it looks good for me" - but as we add some additional padding in this branch, some people/device/dpi will get even more padding that what you added.
And with this additional padding, you'll possibly get one line less of what you'd get without - so, going against what you wished with this PR :)
And we were indeed fine with no padding previously, so we should still be fine without it.

Reply:
I'm fine with removing it. I just added the padding because "it looks good for me", as you said.